app-emulation/aranym/aranym-1.1.0.recipe

```diff
@@ -37,8 +38,16 @@ BUILD_PREREQUIRES="
 
 BUILD()
 {
+	WITHDISDIP=""
+	if [ $effectiveTargetArchitecture == x86_gcc2 ]; then
+		WITHDISDIP="--enable-disdip"
+	fi
+	
 	NO_CONFIGURE=1 ./autogen.sh
-	runConfigure ./configure
+	
+	export CXXFLAGS="$CXXFLAGS -Wno-unused-function -Wno-unused-variable"
+	export CFLAGS="$CFLAGS -Wno-unused-function -Wno-unused-variable"
```

Comment:
I was looking inside the code where it got initialized just to find out it was here ;)
These are not valid for gcc2, without these the build still fails for gcc2